build.gradle

```diff
@@ -0,0 +1,137 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+import java.time.Instant
+import java.time.LocalDate
+
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '2.2.1.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+    id 'com.github.ben-manes.versions' version '0.27.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.50'
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+configurations {
+    agentDeps
+    agentDeps.transitive = false
+}
+
+project.sourceCompatibility = 11
+project.targetCompatibility = 11
+
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+}
+compileTestKotlin {
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+}
+
+group = 'uk.gov.justice.digital.hmpps'
+
+def today = Instant.now()
+def todaysDate = LocalDate.now().format('yyyy-MM-dd')
+version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
+
+jar {
+    baseName = rootProject.name
+    manifest {
+        attributes 'Implementation-Title': rootProject.name, 'Implementation-Version': version
+    }
+}
+
+springBoot {
+    buildInfo {
+        properties {
+            artifact = rootProject.name
+            version = version
+            group = group
+            name = rootProject.name
+            time = today
+            additional = [
+                    by                   : System.properties['user.name'],
+                    operatingSystem      : "${System.properties['os.name']} (${System.properties['os.version']})",
+                    continuousIntegration: System.getenv('CI') ? true : false,
+                    machine              : InetAddress.localHost.hostName
+            ]
+        }
+    }
+}
+
+configurations {
+    implementation.exclude module: 'tomcat-jdbc'
+}
+
+dependencyManagement {
+    imports {
+        mavenBom(SpringBootPlugin.BOM_COORDINATES)
+    }
+}
+
+dependencies {
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+    annotationProcessor 'org.projectlombok:lombok:1.18.10'
+
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    compileOnly 'org.projectlombok:lombok:1.18.10'
+
+    runtime 'com.h2database:h2:1.4.200'
+    runtime 'org.flywaydb:flyway-core:6.0.8'
+    runtime 'org.postgresql:postgresql:42.2.8'
+
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
+    implementation 'org.hibernate:hibernate-java8'
+
+    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-cache'
+
+    implementation 'org.springframework.boot:spring-boot-starter-security'
+    implementation 'org.springframework.security.oauth:spring-security-oauth2:2.3.6.RELEASE'
```

Comment:
worth running `./gradlew dependencyUpdates` before you commit and updating any dependencies that aren't beta - e.g. this should now be on 2.4.0